js/core/tv.js

```diff
@@ -0,0 +1,94 @@
+/*
+ * TV Simulator '99
+ * @Author: zshall 
+ * @Date: 2017-05-02 22:34:58 
+ * @Last Modified by: zshall
+ * @Last Modified time: 2017-05-03 00:31:21
+ */
+const maxWidth = 1200;
+const maxHeight = 1000;
+const warmupTime = 3000;
+var firstStart = true;
+const classes = {
+    Channel12
+};
+
+$(document).ready(() => {
+    if( !Helpers.isMobile().any() ){
+        $(window).resize(function() {
+            handleResize();
+        });
+        $('.screen').addClass('scanlines');
+    }
+
+    handleResize();
+    
+    // about
+    $('#about-button').click(() => {
+        $('.about').toggle();
+    })
+    $('.about').toggle();
+
+    // other TV buttons
+    $('#tv-mute').click(() => {
+        const muted = muteVideo();
+        if (muted) {
+            $('#tvm-bottom-left').text('MUTING');
+        } else {
+            $('#tvm-bottom-left').text('');
+        }
+    });
+    
+    loadYouTubeAPI();
+});
+
+function onYouTubeIframeAPIReady() {
+    // initialize listings grid
+    $.ajax({
+        type: 'GET',
+        url: 'data/guide.xml',
+        dataType: 'xml',
+        complete: (data, status) => {
+            window.guideData = $($.parseXML(data.responseText));
+            var defaultChannel = guideData.find('channel[watchable][default]').attr('number');
+            if (undefined != defaultChannel && null != defaultChannel) {
+                showChannel(defaultChannel);
+            }
+        }
+    });
+}
+
+function showChannel(number, callback) {
```

Comment:
Might want to avoid global functions. At a minimum, attach them to a presumably unique global variable like `programGuide`.